Extensions/ArtifactEngine/Engine/artifactEngine.ts

```diff
@@ -114,8 +115,33 @@ export class ArtifactEngine {
         }
     }
 
+    shouldDownload(path: string) {
```

Comment:
Add a test to Integration tests also (https://github.com/Microsoft/vsts-rm-extensions/blob/master/Extensions/ArtifactEngine/IntegrationTests/jenkinsTests.ts)

Reply:
added tests

Reply:
ok